.zenodo.json

```diff
@@ -1,85 +1,10 @@
 {
-	"description": "The Automated Systematic Review (ASReview) project implements learning algorithms that interactively query the researcher. This way of interactive training is known as Active Learning. ASReview offers support for classical learning algorithms and state-of-the-art learning algorithms like neural networks. The software ships with an user interface as well as a simulation mode.",
-	"title": "ASReview: Active learning for systematic reviews",
+	"description": "The Automated Systematic Review (ASReview) project implements learning algorithms that interactively query the researcher. This way of interactive training is known as Active Learning. ASReview offers support for classical learning algorithms and state-of-the-art learning algorithms like neural networks. ASReview LAB is ASReview LAB is the graphical user interface of the open-source research software and ships with an Oracle, Exploration and Simulation Mode.",
+	"title": "ASReview LAB: Active learning for systematic reviews software",
```

Comment:
Would be nice to align with (the title bar of the tab)
https://github.com/asreview/asreview/blob/623434b022ae0772584c2dbcc0d6ab9492c190f0/asreview/webapp/public/index.html#L11